packages/terra-demographics-banner/src/selectable-demographics-banner/_BackgroundTile.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './SelectableDemographicsBanner.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Child Nodes that are rendered within background tile.
+   */
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  children: '',
```

Comment:
I don't think we should be setting the default to ''. If we don't provide a default, undefined will be used

Reply:
updated : [here](https://github.com/cerner/terra-core/pull/2856/commits/d063a9d511619a4181c537d2030c86f777acf38b#diff-9fd0ae0d0298ecb1d100f96d13178ad4)